src/backend/sgx/mod.rs

```diff
@@ -40,7 +40,8 @@ impl crate::backend::Backend for Backend {
 
     #[inline]
     fn have(&self) -> bool {
-        self.data().iter().all(|x| x.pass)
+        //self.data().iter().all(|x| x.pass)
+        true
```

Comment:
At least, you could have reverted to the last state:
```suggestion
        data::dev_sgx_enclave().pass
```

Reply:
I think that opt-out flag is fine. I'll rework this.